### PR TITLE
chore: added docs on feature toggles

### PR DESCRIPTION
**Pull request by andreabadesso**
You can see it rendered [here](https://github.com/HathorNetwork/hathor-wallet-service/blob/a7b9acea4b8aebca51e117df6bd57f6b90befd44/docs/2022-04-18-feature-toggles.md)

### Acceptance Criteria
- We should have a document describing what are the feature flags the wallet-service uses
- We should have a document explaining how to add a user to the wallet-service beta testing
- We should have a document explaining how to increase or decrease the percentage of public users on the wallet-service beta testing

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: docs/2022-04-18-feature-toggles.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+### Setting a percentage of our user base to use a feature
+
+We have the `gradual rollout strategy` on all of our wallet-service feature toggles, so in order to change the percentage of our total userbase that will receive `true` for the wallet-service facade feature toggle, we need to do the following:
+
+1. Go on the `feature toggles list screen` and find the correct feature flag for the device OS we want to change
+
+![Feature Toggles List](images/feature-toggle-img1.jpg)
+
+2. On the feature toggle details screen, find the `Gradual rollout strategy` and click on the `pen` icon to edit
```
Comment (pedroferreira1) [inline docs/2022-04-18-feature-toggles.md:92]: We ended up up without an image for this step, should we add one?

**Review by pedroferreira1 — APPROVED** [review]

File: docs/2022-04-18-feature-toggles.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+We have a set of strategies configured for each of the feature toggles described above, they are:
+
+
+1. [UserIDs](https://docs.getunleash.io/user_guide/activation_strategy#userids)
+
+Activates for users with a `userId` defined in the `userIds` list. We are currently using unique device identifiers on the mobile wallets (ios and android) and a random identifier on the desktop wallet (that is stored on the device's storage to persist between restarts).
+
+1. [Gradual rollout](https://docs.getunleash.io/user_guide/activation_strategy#gradual-rollout)
+
+This is a `pergentage` based strategy, it will answer the feature toggles depending on the percentage of users that already received a positive or negative answer.
```
Comment (luislhl) [inline docs/2022-04-18-feature-toggles.md]: Typo: `pergentage`

File: docs/2022-04-18-feature-toggles.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+![Feature toggles details screen](images/feature-toggle-img5.jpg)
+***Feature toggles details screen***
+
+Our wallets are constantly polling unleash for feature toggle updates, so on the next poll, if the wallet-service feature toggle changed to `false` and it was `true`, it will trigger a reload, sending the user to the old facade.
+
+### Setting a percentage of our user base to use a feature
+
+We have the `gradual rollout strategy` on all of our wallet-service feature toggles, so in order to change the percentage of our total userbase that will receive `true` for the wallet-service facade feature toggle, we need to do the following:
+
+1. Go on the `feature toggles list screen` and find the correct feature flag for the device OS we want to change
```
Comment (luislhl) [inline docs/2022-04-18-feature-toggles.md:88]: We should have a guide on how to reach this admin interface somewhere.

I think the best place would be in our ops repo, maybe in the main README.md. Also inside the SOP there for the wallet-service would be nice to have the link.

**Review by luislhl — APPROVED** [review]

File: docs/2022-04-18-feature-toggles.md (hunk 1 above)
Comment (andreabadesso) [reply docs/2022-04-18-feature-toggles.md:92]: Added the image on c5b63913ab48880600c4abbc8467131338139ccc

File: docs/2022-04-18-feature-toggles.md (hunk 2 above)
Comment (andreabadesso) [reply docs/2022-04-18-feature-toggles.md]: Fixed in c5b63913ab48880600c4abbc8467131338139ccc

**Pull request merged**